base/applications/mscutils/eventvwr/lang/bg-BG.rc

```diff
@@ -122,13 +122,16 @@ BEGIN
     EDITTEXT IDC_LOGNAME, 67, 23, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
     LTEXT "Log &file:", IDC_STATIC, 7, 41, 60, 8
     EDITTEXT IDC_LOGFILE, 67, 39, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
-    LTEXT "Size:", IDC_STATIC, 7, 57, 60, 8
+// The following 4 IDC_STATIC shall have an accelerator in ReactOS, although MS doesn't have accels for them.
+// Dear Translators: First fill out ALL accels that MS brings as well in this dialog for your language,
```

Comment:
"Dear Translators:" sounds overpolite here. "Note for Translators:"...

Reply:
I acted on your suggestion, I used "Translation note". Resolved.